src/components/RoomDisplay.jsx

```diff
@@ -25,31 +38,45 @@ function RoomDisplay(props) {
     setOpen(true);
   }
 
-  function handleClose() {
-    setOpen(false);
+  function handleClose(boolean) {
```

Comment:
the only change I would make is changing 'boolean' to something like, 'makePostRequest'